code/style.css

```diff
@@ -1,7 +1,194 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+/* styling for all of website*/
 
-.active + .answer {
-  /* some styling here */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  -webkit-font-smoothing: antialiased;
+  text-rendering: optimizeLegibility;
+  font-family: 'Cormorant', serif;  
+  font-family: 'Cormorant Garamond', serif;
+  font-family: 'Spectral', serif;
 }
+
+/* styling for image-header and it's content*/
+
+.image-header {
+  display: flex;
+  justify-content: center;
+  position: relative;
+  height: 55vw;
+  width: 100vw;
+  background-image: url('hero.png');
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+  overflow: hidden;  
+  text-align: center;
```

Comment:
There is no text in the `.header` so `text-align` can be removed :) Your `.title` is a child to `.header` so it is aligned with flex aka `justify-content: center;`